templates/exports/partials/run_history_table.html

```diff
@@ -50,7 +50,7 @@
 
       {% if not is_partial_list %}
       <td>
-        <a class="button is-primary is-rounded is-small" href="/media/{{ export_run.export_config_version.field_dict.config_file }}">
+        <a class="button is-primary is-rounded is-small" href="{{export_run.export_config_version.field_dict.config_file.url}}">
```

Comment:
unfortunately, this is not going to work after https://github.com/dimagi/commcare-sync/pull/24/ is merged. But since this has been such a slog I'm going to recommend we merge this first and then I can address that as part of #24